tests/core/gui/test_file_history.py

```diff
@@ -13,6 +14,7 @@ def catch_file_history_widget(widget_catcher_factory):
 
 @pytest.mark.gui
 @pytest.mark.trio
+@customize_fixtures(real_data_storage=True)
```

Comment:
Could you be more explicit ?

Reply:
we want to keep using the in-memory mock for the pure python test, this is not possible if we put `@customize_fixtures(real_data_storage=True) ` in each test

So instead we should modify the `persistent_mockup` fixture so that it doesn't do anything if it detects the code is currently running with oxidation stuff